### ISSUE TITLE
Terminal loses input after python ran a sub-bash

### ISSUE THREAD

**Issue by mstv** [microsoft/vscode#183419]
Type: <b>Bug</b>

1. create `repro.sh`:
```
#!/usr/bin/bash

echo "### $0 ###"
```

2. create `repro.py`
```
#!/usr/bin/env python3

import subprocess

subprocess.call(['bash', '-c', './repro.sh'])

choice = input(" >> ")
print(choice)
```

3. open a terminal for `bash`
4. run `python repro.py`
5. press `Enter`

--> nothing happens, terminal must be killed

This is not the first affected version. It started to occur several months ago.

VS Code version: Code 1.78.2 (b3e4e68a0bc097f0ae7907b217c1119af9e03435, 2023-05-10T14:39:26.248Z)
OS version: Windows_NT x64 10.0.19045
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz (12 x 3696)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|31.85GB (7.41GB free)|
|Process Argv|d:\\my\\project|
|Screen Reader|no|
|VM|0%|
</details>

<details>
<summary>Extensions (4)</summary>

Extension|Author (truncated)|Version
---|---|---
python|ms-|2023.8.0
vscode-pylance|ms-|2023.5.40
cpptools|ms-|1.15.4
cpptools-extension-pack|ms-|1.3.0

(1 theme extensions excluded)

</details>

**Comment by meganrogge** [comment]
With those steps in Insider's, I see:

![Screenshot 2023-05-25 at 8 37 44 AM](https://github.com/microsoft/vscode/assets/29464607/7a72cbf5-d8f2-424d-bf83-9dd6c1842308)

Can you repro in Insider's? Maybe duplicate of  #181939

**Comment by mstv** [comment]
Your bash has denied to execute `repro.sh`...

**Comment by meganrogge** [comment]
Yep, but that means it's running it IE something is happening as compared with what you reported - nothing happening
